ts-discordjs/klotho.yaml

```diff
@@ -0,0 +1,7 @@
+app: discord-bot
+provider: aws
+path: .
+out_dir: compiled
+defaults:
+    execution_unit:
+        type: fargate
```

Comment:
Is there a reason for fargate specifically?

(Without doing the math, it seems like it would be significantly more expensive.)

Reply:
The discordjs client opens a websocket to discord's API and receives events from it. If this were a lambda, it could not do so as it would eventually timeout in addition to being more expensive than fargate when always-on.

Reply:
ew (on their part, not yours)